modules/distr-tracing-config-otel-send-metrics-monitoring-stack.adoc

```diff
@@ -0,0 +1,40 @@
+////
+This module included in the following assemblies:
+-distr_tracing_install/distributed-tracing-deploying-otel.adoc
+////
+:_content-type: REFERENCE
+[id="distr-tracing-send-metrics-monitoring-stack_{context}"]
+= Sending metrics to the monitoring stack
+
+The following example shows how to configure the monitoring stack to scrape OpenTelemetry collector metrics endpoints and removes
+duplicated labels added by the monitoring stack during scraping.
```

Comment:
Nitpick: instead of introducing a specific example, you could introduce the overall possibility. IMO, the example title is sufficiently clear about it being an example.
```suggestion
You can configure the monitoring stack to scrape OpenTelemetry collector metrics endpoints and to remove duplicated labels that the monitoring stack has added during scraping.
```